src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -592,6 +599,18 @@ uct_ib_mlx5_devx_mem_reg(uct_md_h uct_md, void *address, size_t length,
         goto err_memh_free;
     }
 
+    if ((md->flags & UCT_IB_MLX5_MD_FLAG_MKEY_BY_NAME_RESERVE) &&
+        (flags & UCT_MD_MEM_SYMMETRIC_RKEY)) {
+        /* Best effort */
+        (void)uct_ib_mlx5_devx_reg_ksm_data_contig(
+                md, &memh->mrs[UCT_IB_MR_DEFAULT], 0, address,
+                (memh->super.flags & UCT_IB_MEM_ACCESS_REMOTE_ATOMIC),
+                md->super.mkey_by_name_reserve.base,
+                md->super.mkey_by_name_reserve.base +
+                        (md->super.mkey_by_name_reserve.size / 2) - 1,
```

Comment:
seems like this calc exists in another place?

Reply:
fixed as detailed in previous comment